index.md

```diff
@@ -108,11 +108,11 @@ If you do not already have an Istio-enabled cluster but want to see what Kiali i
 
 ```
 wget https://raw.githubusercontent.com/kiali/kiali/master/hack/cluster-openshift.sh
-sh cluster-openshift.sh --istio-version 0.7.1 --kiali-enabled true up
+sh cluster-openshift.sh --istio-version 0.8.0 --kiali-enabled true up
```

Comment:
--istio-version doesn't need to be specified now - 0.8.0 is the latest and is the default.